test/game_statistics_test.rb

```diff
@@ -144,13 +144,101 @@ def test_count_of_games_by_season
     game4 = MockGame.new(20132014, nil, nil)
     game5 = MockGame.new(20132014, nil, nil)
     game6 = MockGame.new(20142015, nil, nil)
-    mock_games = [game1, game2, game3, game4]
+    mock_games = [game1, game2, game3, game4, game5, game6]
     @stat_tracker.stubs(:games).returns(mock_games)
 
     expected = {"20122013" => 3, "20132014" => 2, "20142015" => 1}
     assert_equal expected, @stat_tracker.count_of_games_by_season
   end
 
+  def test_average_goals_per_game
```

Comment:
These mocks look good, while I don't yet fully understand them, they should speed up the testing!